hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMSecurityProtocolServer.java

```diff
@@ -439,20 +424,18 @@ public String getCACertificate() throws IOException {
   }
 
   /**
-   *
-   * @param role            - node role: OM/SCM/DN.
-   * @param startSerialId   - start certificate serial id.
-   * @param count           - max number of certificates returned in a batch.
-   * @param isRevoked       - whether list for revoked certs only.
+   * @param role          - node role: OM/SCM/DN.
+   * @param startSerialId - start certificate serial id.
+   * @param count         - max number of certificates returned in a batch.
    * @return
    * @throws IOException
    */
   @Override
   public List<String> listCertificate(NodeType role,
-      long startSerialId, int count, boolean isRevoked) throws IOException {
+      long startSerialId, int count) throws IOException {
     List<X509Certificate> certificates =
-        scmCertificateServer.listCertificate(role, startSerialId, count,
-            isRevoked);
+        scmCertificateServer.listCertificate(role, startSerialId, count
+        );
```

Comment:
unnecessary new line here

Reply:
you are right, thank you for spotting this one...